lib/xcodeproj/workspace.rb

```diff
@@ -67,9 +89,36 @@ def self.from_s(xml, workspace_path = '')
     # @return [void]
     #
     def <<(projpath)
```

Comment:
Yes, YARD uses commas to separate the different possible types for a parameter.

For a quick cheat sheet, see http://yardoc.org/types.html